lib/dockerize_rails/docker_commands/helpers.rb

```diff
@@ -0,0 +1,27 @@
+module DockerizeRails
+  module DockerCommands
+    module Helpers
+      NAMES = {
+        image: {
+            rails: "#{DRConfig.application_name}_rails:#{DRConfig.application_env}".freeze,
+            mysql: "#{DRConfig.application_name}_mysql:#{DRConfig.application_env}".freeze,
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.

<!-- Comment by [SideCI](https://sideci.com) -->